gensrc/script/Makefile

```diff
@@ -36,15 +36,6 @@ ${GEN_FUNC_OUTPUT}: gen_functions.py | ${BUILD_DIR}/python
 gen_func: ${GEN_FUNC_OUTPUT}
 .PHONY: gen_func
 
-# generated vector function
-GEN_VEC_FUNC_OUTPUT = ${BUILD_DIR}/python/generated_vector_functions.py \
```

Comment:
so will the 'generated_vector_functions.py' not be used again?

Reply:
> so will the 'generated_vector_functions.py' not be used again?

yes